201-api-management-logs-to-moesif-using-eventhub-webapp/nested/microsoft.eventhub/namespaces.json

```diff
@@ -0,0 +1,156 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "eventHubNsName": {
+            "type": "String"
+        },
+        "eventHubName": {
+            "type": "String"
+        },
+        "eventHubSendPolicyName": {
+            "type": "String",
+            "defaultValue": "evt-hub-apim-send-policy"
+        },
+        "eventHubListenPolicy": {
+            "type": "String",
+            "defaultValue": "evt-hub-app-consume-policy"
+        },
+        "ehNsSkuName": {
+            "type": "String",
+            "defaultValue": "Basic"
+        },
+        "ehNsSkuTier": {
+            "type": "String",
+            "defaultValue": "Basic"
+        },
+        "ehNsSkuCapacity": {
+            "type": "String",
+            "defaultValue": "1"
+        },
+        "ehNsIsAutoInflateEnabled": {
+            "type": "String",
+            "defaultValue": "false"
+        },
+        "ehNsMaximumThroughputUnits": {
+            "type": "String",
+            "defaultValue": "0"
+        },
+        "ehNsZoneRedundant": {
+            "type": "Bool",
+            "defaultValue": false
+        },
+        "messageRetentionInDays": {
+            "type": "int",
+            "defaultValue": 1,
+            "metadata": {
+                "description": "Number of days to retain messages in eventhub"
+            }
+        },
+        "tags": {
+            "type": "Object",
+            "defaultValue": {}
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
```

Comment:
remove defaultValue this must be passed through

Reply:
will update shortly. Thx for catching.